nwbwidgets/test/test_timeseries.py

```diff
@@ -11,3 +11,127 @@ def test_timeseries_widget():
                     rate=100.)
 
     traces_widget(ts)
+
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+from nwbwidgets.base import fig2widget
+
+
+def test_fig2widget():
+    
+    data = np.random.rand(160,3)
+    
+    fig=plt.figure(figsize=(10, 5))
+    plt.plot(data)
+
+    fig2widget(fig)
+
+
+
+
+from pynwb import TimeSeries
+from nwbwidgets.utils.timeseries import get_timeseries_tt
+
+
+def test_get_timeseries_tt():
+    
+    data = list(range(100, 200, 10))
+    ts = TimeSeries(name='test_timeseries', data=data, unit='m', starting_time=0.0, rate=1.0)
+    
+    tt = get_timeseries_tt(ts)
+    np.testing.assert_array_equal(tt,[0., 1., 2., 3., 4., 5., 6., 7., 8., 9.])
+
+    
+    
+
+
+    
+import numpy as np
+from pynwb import NWBFile
+from nwbwidgets.base import processing_module
+from datetime import datetime
+from dateutil.tz import tzlocal
+from pynwb.behavior import Position
+from pynwb.behavior import SpatialSeries
+from pynwb import ProcessingModule
+
+
+def test_processing_module():
```

Comment:
test_processing_module should not be included in this PR because it has nothing to do with timeseries changes and tests

Reply:
What about fig2widget() and vis2widget()? I think they should be moved to another file with another PR.

Reply:
yes, please separate changes into distinct PRs where appropriate